website/content/docs/v0.7/Guides/configuring-pull-through-cache.md

```diff
@@ -14,11 +14,11 @@ It can be also used to verify that all the images are available in local registr
 The follow are requirements for creating the set of caching proxies:
 
 - Docker 18.03 or greater
-- Local cluster requirements for either [docker](docker) or [QEMU](qemu).
+- Local cluster requirements for either [docker](docker) or [fireracker](firecracker).
```

Comment:
this file looks to be reverted. Firecracker and no ghcr io.

Reply:
:+1: